app/assets/javascripts/controllers/ops/pglogical_replication_form_controller.js

```diff
@@ -228,9 +228,10 @@ ManageIQ.angular.app.controller('pglogicalReplicationFormController', ['$http',
       data["user"] = subscription.user;
       data["password"] = subscription.password;
       data["port"]     = subscription.port;
+      data["id"] = subscription.id
     }
     miqService.sparkleOn();
-    var url = '/ops/pglogical_validate_subscription/' + pglogicalReplicationFormId;
+    var url = '/ops/pglogical_validate_subscription'
```

Comment:
@h-kataria if you do not feel comfortable with changes  I will revert it back and will use ``` data["id"] = subscription.id```  to submit subscription id